src/mscorlib/corefx/System/Globalization/CompareInfo.cs

```diff
@@ -379,13 +379,17 @@ public unsafe virtual bool IsPrefix(String source, String prefix, CompareOptions
                     SR.ArgumentNull_String);
             }
             Contract.EndContractBlock();
-            int prefixLen = prefix.Length;
 
-            if (prefixLen == 0)
+            if (prefix.Length == 0)
             {
                 return (true);
             }
 
+            if (source.Length == 0)
```

Comment:
please ignore this comment as I found IsSuffix already have this behavior and we already did that intentionally to be compatible  with behavior of the desktop. please include the change

---
In reply to: [77197668](https://github.com/dotnet/coreclr/pull/7020#discussion_r77197668) [](ancestors = 77197668)